permissionschemes_test.go

```diff
@@ -0,0 +1,56 @@
+package jira
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+func TestPermissionSchemeService_GetList(t *testing.T) {
```

Comment:
Please add negative tests (those that validate when an error should be returned)